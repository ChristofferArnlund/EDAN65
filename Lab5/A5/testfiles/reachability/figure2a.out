reach(fac) = {fac}
reach(f) = {fac, print}
reach(main) = {fac, f, print}
