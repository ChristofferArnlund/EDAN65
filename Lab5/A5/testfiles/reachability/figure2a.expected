reach(fac) = {fac}
reach(f) = {fac, print}
reach(main) = {f, fac, print}
